### PR TITLE
dev-python/nuitka: add 0.6.14

### PR DESCRIPTION
**Pull request by oz123**
Signed-off-by: Oz N Tiram <oz.tiram@gmail.com>

### CODE REVIEW

File: dev-python/nuitka/nuitka-0.6.14.ebuild [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS="rdepend"
+
+PYTHON_COMPAT=( python3_{7,8} )
```
Comment (thesamesam) [inline dev-python/nuitka/nuitka-0.6.14.ebuild:8]: no 3.9? (we're switching next month)

File: dev-python/nuitka/nuitka-0.6.14.ebuild [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+HOMEPAGE="https://www.nuitka.net"
+SRC_URI="https://nuitka.net/releases/${P^}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-util/scons[${PYTHON_USEDEP}]"
```
Comment (thesamesam) [inline dev-python/nuitka/nuitka-0.6.14.ebuild:22]: Definitely?

**Review by thesamesam — CHANGES_REQUESTED** [review]

File: dev-python/nuitka/nuitka-0.6.14.ebuild (hunk 2 above)
Comment (oz123) [reply dev-python/nuitka/nuitka-0.6.14.ebuild:22]: I am not sure what you mean here ...

Comment (thesamesam) [reply dev-python/nuitka/nuitka-0.6.14.ebuild:22]: Scons, being a build system, is a weird runtime dep

File: dev-python/nuitka/nuitka-0.6.14.ebuild (hunk 1 above)
Comment (oz123) [reply dev-python/nuitka/nuitka-0.6.14.ebuild]: That's a good addition! Thanks for reminding me about it. I am already using Python 3.9 ...

File: dev-python/nuitka/nuitka-0.6.14.ebuild (hunk 2 above)
Comment (ionenwks) [reply dev-python/nuitka/nuitka-0.6.14.ebuild:22]: It's normal here.
```
$ nuitka3 --help | grep show-scons
    --show-scons        Operate Scons in non-quiet mode, showing the executed
```

Comment (thesamesam) [reply dev-python/nuitka/nuitka-0.6.14.ebuild:22]: Sure, but having no knowledge of it, it was a reasonable question to ask :)

Comment (oz123) [reply dev-python/nuitka/nuitka-0.6.14.ebuild:22]: Ah, OK. Nuitka is a transpiler compier . It first transpiles the Python code to C, and then compiles it to native code. For the second part it uses scons.

**Comment by juippis** [comment]
9cb54287016c881da592a6efe547f219646df5bc sorry for not getting into this earlier, good news is that python is co-maintaining it, so they'll now get notified about your updates.

**Pull request closed without merging**